codebases/omgevingsbeleid.html

```diff
@@ -0,0 +1,109 @@
+---
+layout: codebase
+title: Codebases we work with - Digital Omgevingsbeleid
+---
+
+<div class="cb-section">
+  <div>
+    <h1 class="title">Codebases we work with</h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-maturity-detail-header">
+    <div class="cb-maturity-detail-title cb-maturity-s1">
+      <h1>Digital Omgevingsbeleid</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s1">Stage 1: <strong>Assessment</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>Digital Omgevingsbeleid was built in response to the 2022 <a href="https://www.government.nl/topics/spatial-planning-and-infrastructure/revision-of-environment-planning-laws">Environment and Planning Act</a> (Omgevingswet) in the Netherlands. Digital Omgevingsbeleid lets you explore laws and policies, view where they apply and compare them to previous versions. This way, policies are more informative and up to date.</p>
+      <p>For example, if someone wants to build a windfarm, they can quickly understand what's allowed on their prospective site. Digital Omgevingsbeleid also makes it easier for a civil servant considering new regulation to see how the new regulations will interact with regulations that already apply.</p>
+    </div>
+
+    <div class="cb-detail-gallery">
+      <a href="#img1">
+          <img class="lb-thumb" src="../photos/codebases/omgevingsbeleid/omgevingsbeleid_thumb_g01.png" alt="Map search in Omgevingsbeleid">
+      </a>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img1">
+        <!--<a href="#img3" class="lb-btn lb-btn-prev"></a>-->
+        <a href="#_" class="lb-btn lb-btn-close"><span class="lb-btn-text">close</span></a>
+        <a href="#img1"><img src="../photos/codebases/omgevingsbeleid/omgevingsbeleid_g01.png" alt="Map search in Omgevingsbeleid"></a>
+        <!--<a href="#img2" class="lb-btn lb-btn-next"></a>-->
+      </div>
+    </div>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We are helping the Omgevingsbeleid team by:</p>
+    <ul>
+      <li>giving advice on open source licenses</li>
+      <li>helping them prepare to publish the code in the open</li>
```

Comment:
Would this bullet from the March newsletter be more clear and specific than this line about preparing?
- "designing workshops on essential skills for open technical and product management of a public codebase."

Reply:
Perhaps, but we have just postponed that one (with an uncertain future).